corelib-storage/src/main/java/eu/europeana/corelib/edm/utils/EdmUtils.java

```diff
@@ -230,7 +230,7 @@ private static void appendTimespans(RDF rdf, List<TimespanImpl> timespans) {
                 addAsObject(timeSpan, Begin.class, ts.getBegin());
                 addAsObject(timeSpan, End.class, ts.getEnd());
                 addAsList(timeSpan, HasPart.class, ts.getDctermsHasPart());
-                addAsList(timeSpan, IsPartOf.class, ts.getIsPartOf());
```

Comment:
please revert back this change and implement the required method in entity api if required.

Reply:
Reverted.